core/src/main/kotlin/api/OrganizationsRoute.kt

```diff
@@ -221,7 +223,7 @@ fun Route.organizations() = route("organizations") {
                     val organizationId = call.requireIdParameter("organizationId")
                     val secretName = call.requireParameter("secretName")
 
-                    secretService.deleteSecretByOrganizationAndName(organizationId, secretName)
+                    secretService.deleteSecret(Entity.ORGANIZATION, organizationId, secretName)
```

Comment:
Similar to the comment on the first commit, it would be nice to have convenience functions like `getRepositorySecret`, `deleteRepositorySecret`, or `listRepositorySecrets`.